test/features/ISECompatibility.test.ts

```diff
@@ -0,0 +1,36 @@
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
+ *--------------------------------------------------------*/
+import * as assert from "assert";
+import * as vscode from "vscode";
+import { ISECompatibilityFeature } from "../../src/features/ISECompatibility";
+
+const ISECompatibility: ISECompatibilityFeature = new ISECompatibilityFeature();
+
+suite("ISECompatibility feature", () => {
+    test("It sets ISE Settings", async () => {
```

Comment:
This one (and only this one) is failing with the message:
```
[0m  1) ISECompatibility feature
       It sets ISE Settings:
[0m[31m     Error: Timeout of 2000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (D:\a\1\s\out\test\features\ISECompatibility.test.js)[0m[90m
      at listOnTimeout (internal/timers.js:531:17)
      at processTimers (internal/timers.js:475:7)
```
this is similar to the message I saw when I first tried getting the tests working on my systems. After a second run the error went away.

However, in the interest of getting this test right, am I reading it right that I should add done() after the async calls in the test?

Reply:
I swear these tests on Windows are so inconsistent. I run it on my Win 10 VM and I get 6 failures all the same as this:
* It sets ISE Settings failed
* It leaves Theme after being changed...
* fails to get latest version
* fails to get latest preview version
* Settings update correctly
* Settings that can only be user settings update correctly

then I run it again immediately... 4 failures this time:
* It sets ISE Settings failed
* It unsets ISE Settings (was passing just a minute ago)
* It leaves Theme after being changed...
* can now get latest version and preview version :man_facepalming: 
* Settings update correctly
* no longer has issue with user settings...